### ISSUE TITLE
Missing file.js on V7.0.0-rc.4

### ISSUE THREAD

**Issue by simoncrypta** [pinojs/pino#1133]
Hey everyone,
I currently work on upgrading Pino to V7 on [RedwoodJS  ](https://github.com/redwoodjs/redwood/pull/3298)and when I use the target `pino/file` for transport  I got this error : 
```
Cannot find module '/home/simon/redwood/node_modules/pino/file.js'
@redwoodjs/api:     Require stack:
@redwoodjs/api:     - /home/simon/redwood/node_modules/pino/lib/worker.js
```

When I wrote myself the `file.js` in `/node_modules/pino/` , everything work as expect. I guess it's just forgotten when deliver to NPM.

**Comment by jsumners** [comment]
Problem is this annoying nonsense -- https://github.com/pinojs/pino/blob/916af455aad308ca980f490ab683dd9f780e130d/package.json#L9-L20

**Comment by mcollina** [comment]
Fixed in v7.0.0-rc.5
